tiledb/sm/subarray/cell_slab.h

```diff
@@ -33,6 +33,7 @@
 #ifndef TILEDB_CELL_SLAB_H
 #define TILEDB_CELL_SLAB_H
 
+#include <cstdint>
```

Comment:
This is a very simple, clean _and plausible_ fix. We had similar changes in the past when new `g++` version tightened the standards on which header files were needed.

Because the world sometimes conspires against us I cannot currently test that as none of the two common ways I sometimes use to get newer compilers seem currently viable:
- Debian has no `gcc-13` yet in unstable or even experimental so no luck there
- Ubuntu tends to have a 'toolchains snapshot' PPA on Launchpad but launchpad.net appears down right now